mmir-plugin-asr-bingWS-web/README.md

```diff
@@ -0,0 +1,62 @@
+# mmir-plugin-asr-nuance-web
+
+Cordova plugin for the MMIR framework that allows Automatic Speech Recognition (ASR) via Bing web services
+
+## configure CSP
+
+(e.g. index.html): allow access to https://dictation.nuancemobility.net
+```
+  <meta http-equiv="Content-Security-Policy"
+        content="default-src 'self' 'unsafe-inline' 'unsafe-eval' https://dictation.nuancemobility.net ...
+```
+
+
+## configuration.json:
+```
+{
+
+...
+
+	    "bingWsWebAudioInput": {
+    		"tokenAddress": "https://api.cognitive.microsoft.com/sts/v1.0/issueToken",
+	    	"webSocketAddress": 			"wss://speech.platform.bing.com/speech/recognition/interactive/cognitiveservices/v1",
+	    	"appKey": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+	    	"encoder": "pcmEncoder.js"
+    },
+	
+	....
+	
+	"mediaManager": {
+    	"plugins": {
+    		"browser": [
+    			...
+                {"ctx": "webn", "mod": "webAudioInput", "config": "webasrBingImpl"},
+                ...
+    		],
+    		"cordova": [
+    			...
+                {"ctx": "webn", "mod": "webAudioInput", "config": "webasrBingImpl"},
+                ...
+    		]
+    	}
+    },
+...
+
+}
+```
+
+## options
```

Comment:
please update this section:  
which general options are supported?
which plugin-specific options can be applied?